internal/daemonset/daemonset.go

```diff
@@ -195,6 +200,25 @@ func (dc *daemonSetGenerator) SetDriverContainerAsDesired(ctx context.Context, d
 		Selector: &metav1.LabelSelector{MatchLabels: standardLabels},
 	}
 
+	if fw := mod.Spec.ModuleLoader.Container.Modprobe.FirmwarePath; fw != "" {
```

Comment:
I don't think that new `if` is covered by a unit test.

Reply:
Good catch. Test added.